### ISSUE TITLE
Code-Skelett implementieren

### ISSUE THREAD

**Issue by Nielsmitie** [Nielsmitie/DM_Lab#4]
- Dataset: Numpy array (n, f), n number of samples, f number of features
        array (n) the class
- Normalize: Function takes in dataset array (n, f) -> array (n, f) z-score normalized
- ID estimation: dataset array (n, f) -> ID value estimation based on the method in the paper
- AE: Build AE with Tensorflow.keras
- Loss function: Redesign the standard Loss function to fit the ones in the paper
1. W
1. S
1. G
- Score function: Build the score function into the existing software
1. W
1. S
1. G
- Model evaluation
